core/src/main/scala/org/apache/spark/status/LiveEntity.scala

```diff
@@ -908,7 +908,7 @@ private[spark] class LiveMiscellaneousProcess(val processId: String,
   var isActive = true
   var totalCores = 0
   val addTime = new Date(creationTime)
-  var removeTime: Date = null
+  val removeTime: Date = null
```

Comment:
It'd be weird if this were always null and never changed. Indeed it looks like this is modified in AppStatusListener?

Reply:
Let me double check this

Reply:
@srowen 

There are 2 `removeTime` in `LiveEntity.scala`:

one is defined in `LiveExecutor`

https://github.com/apache/spark/blob/feba5ac32f2598f6ca8a274850934106be0db64d/core/src/main/scala/org/apache/spark/status/LiveEntity.scala#L262-L272

and the other one defined in `LiveMiscellaneousProcess`

https://github.com/apache/spark/blob/feba5ac32f2598f6ca8a274850934106be0db64d/core/src/main/scala/org/apache/spark/status/LiveEntity.scala#L904-L912

The `removeTime`  defined in `LiveExecutor` is modified in `AppStatusListener` but this PR does not change its definition.

This PR change the definition of  `removeTime`  in `LiveMiscellaneousProcess` because it has not been modified

Reply:
Ah OK I'm looking at the wrong one. Can we just remove this and replace with 'null' in the class then? seems like no purpose in declaring it then

Reply:
done